.github/workflows/test-backend.yml

```
@@ -50,8 +50,16 @@ jobs:
       uses: pnpm/action-setup@v4
       with:
         run_install: false
+    - name: Cache FFmpeg binaries
+      uses: actions/cache@v4
+      with:
+        path: /opt/hostedtoolcache/ffmpeg
+        key: ${{ runner.os }}-${{ runner.arch }}-ffmpeg-${{ step.ffmpeg.outputs.ffmpeg-version }}
```

actionlint (1.7.4)

undefined variable "step". available variables are "env", "github", "inputs", "job", "matrix", "needs", "runner", "secrets", "steps", "strategy", "vars"

(expression)